inspirehep/modules/theme/templates/inspirehep_theme/accounts/forgot_password.html

```diff
@@ -0,0 +1,49 @@
+{#
+## This file is part of INSPIRE.
+## Copyright (C) 2016 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% extends config.SEARCH_UI_BASE_TEMPLATE %}
```

Comment:
Better to extend from `inspirehep_theme/page.html` since we don't install `invenio_search_ui` - that config variable might disappear.